package.json

```diff
@@ -8,12 +8,8 @@
     "format": "prettier --write \"**/*.{ts,tsx,js,jsx,json}\"",
     "test": "cd ./react && yarn test",
     "verify": "yarn lint && yarn test",
-    "lint:locales": "intl-equalizer"
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged"
-    }
```

Comment:
you need add the hooks 
https://github.com/typicode/husky#usage

Reply:
No, it just needs a rebase as @lariciamota said above. The changes on husky were done at other PR

Reply:
Hm, okay. So, someone need to make the rebase